uber/static/styles/styles.css

```diff
@@ -1,8 +1,16 @@
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 14px;
+    line-height: 1.42857143;
+    color: #333;
+    background-color: #fff
+}
```

Comment:
Is this related to this change or did this get pulled in from another branch?  I only ask because changing the default font settings seems like a doozy, though I'm sure you've tested this enough that it's fine.

Reply:
I am not a fan of times new roman in my web browser.

Reply:
Haha, fair enough, I do recall that the Helvetica documentary made a big impact :P

Reply:
That's snippet is a copy/paste from the bootstrap css we use throughout the app. I wanted to maintain _some_ kind of consistency between the pages that use the bootstrap UI, and legacy pages which don't. 

Although I haven't audited the code lately and we may have updated all the legacy pages by now. I added this because I ran into one page that wasn't using bootstrap for this pull request (I updated that page, btw).